packages/fx-core/src/plugins/resource/appstudio/appStudio.ts

```diff
@@ -52,39 +52,6 @@ export namespace AppStudioClient {
     }
   }
 
-  /**
```

Comment:
Remove this method, currently we're using the implementation in solution/appstudio.